### ISSUE TITLE
API: Remove `featured` property from News model

### ISSUE THREAD

**Issue by gideonthomas** [mozilla/foundation.mozilla.org#721]
Since we're switching to an admin managed homepage which deals with the relationship between the homepage and people, news, and highlights using foreign key relationships, there is no need for the `featured` property that we currently have in the News and Person models.

We would need to note down the currently featured news and person instances so that we can migrate/translate that data over to the new admin managed homepage (which does not use this `featured` flag).

The front-end would also need to be updated to not rely on the `feature` flag of these two models before we can remove this.

**Edit by Alan: For this ticket, let's just remove it from the News model. The People model will eventually entirely change.**

**Comment by kristinashu** [comment]
Is this still relevant now that we're using Wagtail?

**Comment by gideonthomas** [comment]
@alanmoo ^

**Comment by alanmoo** [comment]
For people, I believe we still need to keep the "featured" flag because it determines if a bio is full-width on the people page. I believe we will eventually be moving to all "people" managed by pulse, at which point the people model as a whole will likely disappear.

For news, I believe we can remove it entirely, no need to preserve it. That model will likely continue to live on and evolve.

**Comment by gideonthomas** [comment]
since we're pulling profiles from pulse now, is this issue still a thing?

**Comment by alanmoo** [comment]
Technically this might break https://foundation.mozilla.org/en/people/, though I'm betting we can just take that page down. @jessevondoom any thoughts?

**Comment by gideonthomas** [comment]
@alanmoo to followup with @jessevondoom and decide the direction for this ticket.

**Comment by alanmoo** [comment]
I'm attempting to remove the featured property from `News`

**Comment by Pomax** [comment]
Technically it didn't - let's do this again, but using the new multi-step approach to database changes.

**Comment by Pomax** [comment]
reopening, https://github.com/mozilla/foundation.mozilla.org/pull/2861 broke prod and needs to be redone. https://github.com/mozilla/foundation.mozilla.org/pull/2981 backs it out, and a new PR needs to be filed.

**Comment by kristinashu** [comment]
😢

**Comment by alanmoo** [comment]
What failed about our approach this time?

**Comment by Pomax** [comment]
closing in favour of https://github.com/mozilla/foundation.mozilla.org/issues/3379
